reportUploadTask/index.ts

```diff
@@ -38,8 +51,10 @@ async function convertCoverageFiles(coverageFiles: string[]): Promise<string> {
 	const outputXmlFile = path.join(path.dirname(coverageFiles[0]), 'coverage.xml');
 
 	let codeCoverageExePath: string = task.getInput('codeCoverageExePath');
-	if (!codeCoverageExePath || codeCoverageExePath.trim() === '') {
+	if (utils.isNullOrEmpty(codeCoverageExePath)) {
```

Comment:
Probably because it is only used here and I wanted to keep it local or simply forgot it ;-). But you're right, that's a bit unexpected :-) I added it to the `TaskParameters`.